<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntEspecificaciones</class>
 <widget class="QDialog" name="IntEspecificaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipos de Especificaciones:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>106</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Seguimiento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Estabilidad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Ruido Sensor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>RPS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>RPE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>EC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>253</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="cons">
      <property name="text">
       <string>Constante</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fun">
      <property name="text">
       <string>Funci√≥n Transferencia</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Frecuencias: [</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="frecini"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="frecfin"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="principal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <widget class="QWidget" name="principal_todos">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QWidget" name="vacia"/>
     <widget class="QWidget" name="constante">
      <widget class="QWidget" name="layoutWidget1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>174</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Altura:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="altura"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>153</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QRadioButton" name="lineal">
          <property name="text">
           <string>Lineal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="decibelios">
          <property name="text">
           <string>Decibelios</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="FT">
      <widget class="QStackedWidget" name="imagenes">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>141</width>
         <height>80</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_9"/>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="imagCP">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5">
        <widget class="QLabel" name="imagLF">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="imagHF">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>219</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Numerador:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nume"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Denominador:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="deno"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>157</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>K:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="k"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ret:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ret"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>216</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="poli">
            <property name="text">
             <string>C. de Polinomios</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="libre">
            <property name="text">
             <string>Formato Libre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="hfgain">
            <property name="text">
             <string>K = hfgain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lfgain">
            <property name="text">
             <string>k = lfgain</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6"/>
    </widget>
    <widget class="QLabel" name="imagen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="principal_se">
    <widget class="QLabel" name="imagen2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="Se_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QWidget" name="vacia_se1"/>
     <widget class="QWidget" name="constante_se1">
      <widget class="QWidget" name="layoutWidget7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>175</width>
         <height>60</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Altura:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="altura_se1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QRadioButton" name="linea_se1">
            <property name="text">
             <string>Lineal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="decibelios_se1">
            <property name="text">
             <string>Decibelios</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="ft_se1">
      <widget class="QWidget" name="layoutWidget8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>191</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="CPoliSe1">
            <property name="text">
             <string>C. polinomios</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="FLSe1">
            <property name="text">
             <string>Formato libre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QRadioButton" name="KGSe1">
            <property name="text">
             <string>k=hfgain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="KNGSe1">
            <property name="text">
             <string>k=lfgain</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>220</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Numerador: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NumeSe1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Denominador: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DenoSe1"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>111</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>K: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="KSe1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Ret:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RetSe1"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="imagen_se_1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>159</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_7">
        <widget class="QLabel" name="imgKGse1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_8">
        <widget class="QLabel" name="imgKNGse1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="imgCPse1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="Se_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QWidget" name="vacia_se2"/>
     <widget class="QWidget" name="constante_se2">
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>175</width>
         <height>60</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Altura:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="altura_se1_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QRadioButton" name="linea_se1_2">
            <property name="text">
             <string>Lineal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="decibelios_se1_2">
            <property name="text">
             <string>Decibelios</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="ft_se2">
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>191</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QRadioButton" name="CPoliSe1_2">
            <property name="text">
             <string>C. polinomios</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="FLSe1_2">
            <property name="text">
             <string>Formato libre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QRadioButton" name="KGSe1_2">
            <property name="text">
             <string>k=hfgain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="KNGSe1_2">
            <property name="text">
             <string>k=lfgain</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>220</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Numerador: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NumeSe1_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Denominador: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DenoSe1_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>K: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="KSe1_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Ret:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RetSe1_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="imagen_se_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_10"/>
       <widget class="QWidget" name="page_11">
        <widget class="QLabel" name="imgKGse1_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_12">
        <widget class="QLabel" name="imgKNGse1_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_13">
        <widget class="QLabel" name="imgCPse1_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>layoutWidget4</zorder>
  <zorder>layoutWidget5</zorder>
  <zorder>layoutWidget6</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>principal</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
